examples/rules_ios/xcodeproj_targets.bzl

```diff
@@ -0,0 +1,23 @@
+"""Exposes targets used by `xcodeproj` to allow use in fixture tests."""
+
+load(
+    "@rules_xcodeproj//xcodeproj:defs.bzl",
+    "top_level_targets",
+)
+
+XCODEPROJ_TARGETS = [
+    top_level_targets(
+        labels = [
+            "//iOSApp",
+            "//Lib:LibDynamic",
+        ],
+        target_environments = ["simulator"],
```

Comment:
This is the default, do you want `device` here as well? I would assume so to show off device support for rules_ios?

Reply:
I had left it off to avoid provisioning profile stuff. Happy to include it if you think it's valuable. That stuff just ends up forwarded to the underlying `rules_apple` things so there's likely not a ton of benefit explicitly WRT `rules_ios` functionality.

Reply:
I would like it, yes.

Reply:
Yep, there's not much going on as far as I remember in `rules_ios` re "on device" attributes. But +1 to adding it anyways if it's simple enough, catching potential future regressions being one benefit here IMO.